.github/CODEOWNERS

```diff
@@ -1,47 +1,78 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
 # https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
 
+################
+# Orphaned paths
+################
+# As of 1/30/2023 these paths have no owners:
+
+# /
+# /.devcontainer/
+# /.vscode/
+# /common/perf-test-score/
+# /doc/
+# /samples/
+
+###########
```

Comment:
Why did this get moved up top? Also why haven't we removed `**/ci.yml` and `**/tests.yml` paths?

Reply:
Fixed by:
- https://github.com/Azure/azure-sdk-for-java/pull/33309